src/app/shared/components/forms/custom-button/custom-button.component.scss

```diff
@@ -29,3 +35,35 @@ button {
   font-weight: 400;
   padding: 1.1rem 3rem;
 }
+
+.register-user {
+  padding: 1rem 2.8rem;
+  font-weight: 700;
+  font-size: 1.8rem;
+  line-height: 2.7rem;
+  text-align: center;
+  color: $color-white;
+}
+
+.medium {
+  padding: 1.7rem 3.1rem;
+  min-width: 15.1rem;
+  justify-content: center;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2rem;
+  line-height: 2.1rem;
+  text-align: center;
+  color: $color-white;
+}
+
+.register-btn {
```

Comment:
what is this. we already have small medium large. do not keep creating buttons for specific cases